File: src/main/java/com/appsolute/soomapi/global/env/property/JwtProperty.java
package com.appsolute.soomapi.global.env.property;

import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.context.annotation.Configuration;

@Configuration
@ConfigurationProperties("jwt")
@Getter @Setter
public class JwtProperty {
    private String secret;
    private Long expiredAt;
}

File: src/main/kotlin/com/appsolute/soomapi/global/env/property/JwtProperty.kt
package com.appsolute.soomapi.global.env.property

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.boot.context.properties.ConstructorBinding

@ConstructorBinding
@ConfigurationProperties("jwt")
data class JwtProperty (
    val secret: String,
    val accessExpiredAt: Long,
    val refreshExpiredAt: Long
)
